.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: macOS-latest
+    
+    steps:
+    - uses: actions/checkout@master
+    - name: Installing Yarn dependencies
+      run: yarn install
```

Comment:
Following [yarn install docs](https://classic.yarnpkg.com/en/docs/cli/install/), a better approach for CI systems may be to call `yarn install` with the `--frozen-lockfile` flag.

Reply:
Sounds good.   I'm also going to remove the redundant build step.   The test step will build and run at the same time